src/python/pants/backend/codegen/protobuf/java/subsystem.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from pants.jvm.resolve.jvm_tool import JvmToolBase
+from pants.util.docutil import git_url
+
+
+class JavaProtobufGrpcSubsystem(JvmToolBase):
+    options_scope = "protobuf-java-grpc"
+    help = "gRPC support for Java Protobuf (https://github.com/grpc/grpc-java)"
+
+    default_version = "1.48.0"
+    default_artifacts = (
+        "io.grpc:protoc-gen-grpc-java:exe:linux-aarch_64:{version}",
+        "io.grpc:protoc-gen-grpc-java:exe:linux-x86_64:{version}",
+        "io.grpc:protoc-gen-grpc-java:exe:osx-aarch_64:{version}",
+        "io.grpc:protoc-gen-grpc-java:exe:osx-x86_64:{version}",
```

Comment:
This is a non-standard use of `JvmToolBase` since the artifacts here on not jars, but rather native binaries.